### PR TITLE
Prompt before deleting component

### LINKED ISSUE #200: delete a component should always ask for confirmation
When deleting a component, we need to ask for confirmation to the user. A --force flag could be introduced as an override of this, to automate tasks.

```
$ ocd delete
Do you really want to delete myfrontend in application <name> (y/n)?
```

Subtask of #156

### PR DESCRIPTION
**Pull request by containscafeine**
The user will be prompted before deletion of a component, unless
overriden using --force/-f.

Fixes #200

### CODE REVIEW

File: cmd/component.go [hunk 1]
```diff
@@ -110,15 +111,26 @@ var componentDeleteCmd = &cobra.Command{
...
 
-		// no flag was set, create empty component
-		output, err := component.Delete(componentName)
-		if err != nil {
-			fmt.Println(err)
-			os.Exit(1)
+		if componentForceDeleteFlag {
+			confirmDeletion = "y"
+		} else {
+			fmt.Printf("Are you sure you want to delete the component: %v? [y/N] ", componentName)
```
Comment (kadel) [inline cmd/component.go]: It should display from what application the component will be deleted.

**Comment by kadel** [comment]
ping @containscafeine

File: cmd/component.go (hunk 1 above)
Comment (containscafeine) [reply cmd/component.go]: fixed

File: cmd/component.go [hunk 2]
```diff
@@ -110,15 +111,26 @@ var componentDeleteCmd = &cobra.Command{
...
+			fmt.Scanln(&confirmDeletion)
 		}
-		fmt.Println(output)
 
+		if strings.ToLower(confirmDeletion) == "y" {
+			// no flag was set, create empty component
```
[the remark this answers is not in the document]
Comment (containscafeine) [reply cmd/component.go]: removed

**Pull request merged**